libs/execution_context/ebpf_link.c

```diff
@@ -555,7 +534,7 @@ _ebpf_link_instance_invoke_batch(
     ebpf_result_t return_value = EBPF_SUCCESS;
```

Comment:
```suggestion
    ebpf_result_t return_value;
```
nit: style consistency with line 459, 491, etc. since it's now initialized at line 537